docs/snapshotting/snapshot-support.md

```diff
@@ -116,13 +116,8 @@ all [supported platforms](../../README.md#tested-platforms).
 
 ### Developer preview status
 
-The snapshot functionality is still in developer preview due to the following:
-
-- Poor entropy and replayable randomness when resuming multiple microvms from
-  the same snapshot. We do not recommend to use snapshotting in production if
-  there is no mechanism to guarantee proper secrecy and uniqueness between
-  guests. Please see
-  [Snapshot security and uniqueness](#snapshot-security-and-uniqueness).
+Diff snapshots are still in developer preview while we are diving deep into how
+the feature can be combined with secret hiding support in Firecracker.
```

Comment:
I know I'm being annoying, but I don't think we should refer to "secret hiding" here, because its far from being in firecracker main, and also because I'm still holding our hope that we won't be calling it secret hiding xP

Reply:
Fine by me, but that's the reason why we're keeping diff snapshots in Dev Preview. So, any suggestions on how to phrase it?

Reply:
"... into how the feature can be combined with other upcoming Firecracker features" maybe? :thinking:

Reply:
IMO, that's quite ambiguous/cryptic. @kalyazin ideas?

Reply:
we can replace "secret hiding" with guest_memfd to stay naming-agnostic.